src/settings.py

```diff
@@ -29,6 +29,14 @@ class Settings(BaseSettings):
     BOT_TOKEN: str
     BOT_WEBHOOK_MODE: bool = False
 
+    # Настройки логирования
+    LOG_DIR: str | Path = BASE_DIR / "logs"
+    LOG_FILE: str = "app.log"
+    LOG_LEVEL: str = "INFO"
+    LOG_FILE_SIZE: int = 10 * 1024 * 1024
```

Comment:
Это может показаться странным, но может лучше так?) 
```suggestion
    LOG_FILE_SIZE: int = 10 * 2**20
```